discovery-provider/src/tasks/vacuum_db.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import time
+from src.tasks.celery_app import celery
+
+logger = logging.getLogger(__name__)
+
+
+@celery.task(name="vacuum_db", bind=True)
+def vacuum_db(self):
+    """Vacuum the db"""
+
+    db = vacuum_db.db
+    redis = vacuum_db.redis
+
+    have_lock = False
+    update_lock = redis.lock("vacuum_db", timeout=3600)
+
+    try:
+        have_lock = update_lock.acquire(blocking=False)
+
+        if have_lock:
+            with db.scoped_session() as session:
+                session.execute("VACUUM FULL ANALYZE")
```

Comment:
we definitely do not want vacuum full. this would claim exclusive locks on the table so nothing can be be written while the rebuild occurs. we just want `VACUUM ANALYZE`